### PR TITLE
Added constants for search API

### PR DESCRIPTION
**Pull request by valentingregoire**
I added some constants for the search API. I used the official search API docs as reference: https://docs.gitlab.com/ee/api/search.html#project-search-api
It's always nice to be able to link to existing constants instead of having to manually foresee strings.

### CODE REVIEW

File: gitlab/const.py [hunk 1]
```diff
@@ -33,3 +33,40 @@
...
+SEARCH_SCOPE_GLOBAL_ISSUES = _SEARCH_SCOPE_ISSUES
+SEARCH_SCOPE_GLOBAL_MERGE_REQUESTS = _SEARCH_SCOPE_MERGE_REQUESTS
+SEARCH_SCOPE_GLOBAL_MILESTONES = _SEARCH_SCOPE_MILESTONES
+SEARCH_SCOPE_GLOBAL_SNIPPET_TITLES = "snippet_titles"
+SEARCH_SCOPE_GLOBAL_WIKI_BLOBS = _SEARCH_SCOPE_WIKI_BLOBS
+SEARCH_SCOPE_GLOBAL_COMMITS = _SEARCH_SCOPE_COMMITS
+SEARCH_SCOPE_GLOBAL_BLOBS = _SEARCH_SCOPE_BLOBS
+SEARCH_SCOPE_GLOBAL_USERS = _SEARCH_SCOPE_USERS
+
+SEARCH_SCOPE_GROUP_PROJECTS = _SEARCH_SCOPE_PROJECTS
```
Comment (max-wittig) [inline gitlab/const.py:56]: Why is the duplication needed?

**Comment by max-wittig** [comment]
Thanks for the MR! Just a question.

**Comment by valentingregoire** [comment]
@max-wittig, any updates on this matter?

**Comment by max-wittig** [comment]
Hey @valentingregoire! Sorry for the delay! Could you maybe add some docs why or how those variables should be used?

**Review by valentingregoire — COMMENTED** [review]
> Hey @valentingregoire! Sorry for the delay! Could you maybe add some docs why or how those variables should be used?

@max-wittig, I have added documentation (and fixed a typo in the meantime).

File: gitlab/const.py (hunk 1 above)
Comment (nejch) [reply gitlab/const.py:57]: Personally, +1 vote for this approach, I like your last suggestion in this comment, it's much more concise than in the original commit :)

**Comment by valentingregoire** [comment]
@max-wittig, any updates?

**Comment by max-wittig** [comment]
All good 👍

**Pull request merged**